src/components/multi_select_state_utils.js

```diff
@@ -1,18 +1,36 @@
-export const findItemByIds = ids => item =>
-  ids.find(id => id === item.id) === undefined;
+const matchItemById = id => item => id === item.id;
 
-export const getSelectAllItems = (filteredItems, selectedItems, items) => {
-  const sourceItems = items.filter(
+const getItemById = (id, selectedItems) =>
```

Comment:
make it 
`const findItem = ({id}, items) => `
we may export it and reuse allover the multi_select_state, because this logic is duplicated over 6 times